src/components/WideRHPContextProvider/getVisibleRHPRouteKeys.ts

```diff
@@ -0,0 +1,30 @@
+import extractNavigationKeys from '@libs/Navigation/helpers/extractNavigationKeys';
+import getLastVisibleRHPRouteKey from '@libs/Navigation/helpers/getLastVisibleRHPRouteKey';
+import {navigationRef} from '@libs/Navigation/Navigation';
+
+/**
+ * Extracts the keys of the screens that are currently displayed from the array of all Wide/Super Wide RHP keys
+ *
+ * @param allWideRHPKeys - an array of all Wide/Super Wide RHP keys
+ */
+function getVisibleWideRHPKeys(allWideRHPKeys: string[]) {
+    const rootState = navigationRef.getRootState();
```

Comment:
We should check if the navigation is ready before using it:

https://github.com/Expensify/App/issues/76536#issuecomment-3602719918